### DISCUSSION TITLE
How to set WinUI 3 Application Window Title?

### DISCUSSION THREAD

**Discussion by greatoceansoftware** [unoplatform/uno#8748] [Q&A]
I've started an Uno WinUI 3 project in VS 2022 that will target the Windows desktop and WebAssembly. The main window (running as a Windows project) simply says "WinUI Desktop" with a generic icon. How can these be changed
![2022-05-11_08-59-47](https://user-images.githubusercontent.com/78753788/167895101-40d41b53-c4e1-4c3f-94ca-abd568edf594.png)
?

**Comment by Daoting** [comment]
https://github.com/microsoft/WindowsAppSDK/issues/1914

**Reply by ApoorvaPetkar** [reply]
@Daoting, the link you posted was about changing the icon, how can we set the title? 

@greatoceansoftware any fix did you find?

**Comment by cckang** [comment]
Can change by adding the title property to MainWIndow class 

 ```
public sealed partial class MainWindow : Window
    {
        public MainWindow()
        {
            this.InitializeComponent();
            Title = "App Name";
        }
    }
```
